pkg/onepassword/items.go

```diff
@@ -30,6 +30,14 @@ func GetOnePasswordItemByPath(opConnectClient connect.Client, path string) (*one
 	if err != nil {
 		return nil, err
 	}
+
+	for _, file := range item.Files {
+		_, err := opConnectClient.GetFileContent(file)
```

Comment:
I assume the file size limit imposed by Connect will also work well here so I will not enforce any restrictions in the operator itself.